src/scripts/createDashboard.ts

```diff
@@ -42,22 +42,113 @@ const ALERT_MARKERS = {
         "alert": "y > 5.8",
     },
 };
+const ALERT_MARKERS_QUERY_VALUE = {
+    'largest-contentful-paint': [
+        {
+            "comparator": ">",
+            "value": 4000,
+            "palette": "white_on_red"
+        },
+        {
+            "comparator": ">=",
+            "value": 2500,
+            "palette": "white_on_yellow"
+        },
+        {
+            "comparator": "<",
+            "value": 2500,
+            "palette": "white_on_green"
+        }
+    ],
+    'first-contentful-paint': [
+        {
+            "comparator": ">",
+            "value": 3,
+            "palette": "white_on_red"
+        },
+        {
+            "comparator": ">=",
+            "value": 1.8,
+            "palette": "white_on_yellow"
+        },
+        {
+            "comparator": "<",
+            "value": 1.8,
+            "palette": "white_on_green"
+        }
+    ],
+    'cumulative-layout-shift': [
+        {
+            "comparator": ">",
+            "value": 0.25,
+            "palette": "white_on_red"
+        },
+        {
+            "comparator": ">=",
+            "value": 0.1,
+            "palette": "white_on_yellow"
+        },
+        {
+            "comparator": "<",
+            "value": 0.1,
+            "palette": "white_on_green"
+        }
+    ],
+    'total-blocking-time': [
+        {
+            "comparator": ">",
+            "value": 600,
+            "palette": "white_on_red"
+        },
+        {
+            "comparator": ">=",
+            "value": 200,
+            "palette": "white_on_yellow"
+        },
+        {
+            "comparator": "<",
+            "value": 200,
+            "palette": "white_on_green"
+        }
+    ],
+    'speed-index': [
+        {
+            "comparator": ">",
+            "value": 5.8,
+            "palette": "white_on_red"
+        },
+        {
+            "comparator": ">=",
+            "value": 3.4,
+            "palette": "white_on_yellow"
+        },
+        {
+            "comparator": "<",
+            "value": 3.4,
+            "palette": "white_on_green"
+        }
+    ],
+};
 
-function fetchAlertMarkersForAudit(auditName: string) {
-    return ALERT_MARKERS.hasOwnProperty(auditName) ? [
+function fetchTimeseriesAlertMarkersForAudit(auditName: string) {
+    return ALERT_MARKERS_TIMESERIES.hasOwnProperty(auditName) ? [
         {
             "label": "Alert",
-            "value": ALERT_MARKERS[auditName].alert,
+            "value": ALERT_MARKERS_TIMESERIES[auditName].alert,
             "display_type": "error dashed"
         },
         {
             "label": "Warning",
-            "value": ALERT_MARKERS[auditName].warning,
+            "value": ALERT_MARKERS_TIMESERIES[auditName].warning,
             "display_type": "warning dashed"
         }
     ] : []
 }
 
+function fetchQueryValueAlertMarkersForAudit(auditName: string) {
+    return ALERT_MARKERS_QUERY_VALUE.hasOwnProperty(auditName) ? ALERT_MARKERS_QUERY_VALUE[auditName] : []
```

Comment:
I see us using `hasOwnProperty` in a few places. Can't we simplify this?

```
return ALERT_MARKERS_QUERY_VALUE[auditName] ?? []

Reply:
TBH, this whole script could be improved and simplified. I thought about refactoring it by using a Factory design pattern but that is quite the over haul for the scope of the issue. 

https://github.com/iFixit/vigilo/compare/main...testing-factory-pattern-for-create-dashboards-script